### ISSUE TITLE
Dock is not visible using joint screen mode

### ISSUE THREAD

**Issue by mauriciobc** [icedman/anino-dock#8]
Fedora 38, Wayland, NVIDIA driver 535.54.03 NVIDIA GeForce RTX 3060 Laptop GPU
![Captura de tela de 2023-07-26 23-05-06](https://github.com/icedman/anino-dock/assets/5098821/9d7fd6ae-195d-4f6c-9e48-9c0e3da99e64)

With an external monitor on, when using "Join displays" configuration, the dock became invisible, but labels appeared when hovered over icons.

**Comment by FelixFourcolor** [comment]
I don't think @icedman still maintains this. :( Which is unfortunate because it's my favorite dock extension.

**Comment by mauriciobc** [comment]
Yeah, that's sad news. I just installed for testing and it's amazing.

**Comment by Ticklezz** [comment]
I hope he comes back, he was working very hard on this and now has been absent for awhile. He might just be busy with real life.
